strawberryfields/engine.py

```diff
@@ -34,7 +35,14 @@
 # for automodapi, do not include the classes that should appear under the top-level strawberryfields namespace
 __all__ = ["BaseEngine", "LocalEngine"]
 
+
+handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+
 log = logging.getLogger(__name__)
+log.addHandler(handler)
+log.setLevel(logging.INFO)
```

Comment:
It's typically bad practice to set a logging handler within a Python library. This is because it will overwrite a users logging handler; this happened in the past in PennyLane! See https://github.com/XanaduAI/pennylane/issues/207.

For more details: https://stackoverflow.com/questions/49631240/python-logging-inheritance-in-pip-installed-packages

Reply:
> One of the points of the logging module is to totally separate logger's usage (instanciation and calls) from logging's config (setting up handlers, formatters, levels, propagation etc). Library code is not supposed to do any configuration - it only uses loggers. Configuring the logging is the application code responsability, and can range from merely configuring the root logger to print to sys.stderr to highly sophisticated configuration with different handlers / levels per sub-logger, dispatching to different outputs (syslog, emails, sentry etc) according to the level and/or sub-logger.

Reply:
Oh, that's really helpful, thanks for that!

Reply:
No worries! If this is something that we want to provide as a standard output, but also something we want to log, it could be the case that we have both a print statement and a `log.info()` call.

Reply:
In that case, I'd say going with simple `print` statement could be something we'd want in the short term. The reason is, that from what I found, we'd have to have a `StreamHandler` to be able to write to the standard output (or at least call `logging.basicConfig`, but that would also ignore the user's logger).

In the long term setting it up with logging and providing a basic introduction to logging could be the way.

Reply:
in my opinion, logging.info is the correct way to do this. If people want these messges, they should use `logging.basicConfig` accordingly. The warning will be printed by default (e.g. in a jupyter notebook).

Reply:
And yes, I agree there should be no logging config being done in SF by default.

Reply:
Added an integration test that contains the user logging configuration that was reported in the PL case.

Reply:
Thanks Antal!